From 779ab48a75875ce0aa1d9aa98991e3dc700bf83b Mon Sep 17 00:00:00 2001
From: vagrant <vagrant@cockpit.vm>
Date: Mon, 24 Aug 2020 12:33:07 +0200
Subject: [PATCH 1/4] Revert "build: Bump gnutls-devel build dependency"

This reverts commit 2351ae5d4c5608e48f1c941128758e5318836eba.

Conflicts:
	src/tls/test-server.c
	tools/debian/control
---
 configure.ac       | 2 +-
 tools/cockpit.spec | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 3a31cb731..a7af127a4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -73,7 +73,7 @@ GIO_REQUIREMENT="gio-unix-2.0 >= $GIO_VERSION gio-2.0 >= $GIO_VERSION"
 LIBSYSTEMD_REQUIREMENT="libsystemd >= 235"
 JSON_GLIB_REQUIREMENT="json-glib-1.0 >= 0.14.0"
 POLKIT_REQUIREMENT="polkit-agent-1 >= 0.105"
-GNUTLS_REQUIREMENT="gnutls >= 3.4.3"
+GNUTLS_REQUIREMENT="gnutls >= 3.3.0"
 KRB5_REQUIREMENT="krb5-gssapi >= 1.11 krb5 >= 1.11"
 
 PKG_CHECK_MODULES(GIO, [$GIO_REQUIREMENT])
diff --git a/tools/cockpit.spec b/tools/cockpit.spec
index 57b6b3e7c..8bb3a59df 100644
--- a/tools/cockpit.spec
+++ b/tools/cockpit.spec
@@ -75,7 +75,7 @@ BuildRequires: gettext >= 0.19.7
 BuildRequires: libssh-devel >= 0.8
 %endif
 BuildRequires: openssl-devel
-BuildRequires: gnutls-devel >= 3.4.3
+BuildRequires: gnutls-devel
 BuildRequires: zlib-devel
 BuildRequires: krb5-devel >= 1.11
 BuildRequires: libxslt-devel
-- 
2.28.0

